source/iofs/README.md

```diff
@@ -0,0 +1,38 @@
+# iofs
+
+Driver with file system interface (`io/fs#FS`) supported from Go 1.16.
+
+This Driver cannot be used with Go versions 1.15 and below.
```

Comment:
suggestion: Reword this to something like just:

> Package iofs provides a Go 1.16+ [`io/fs`](https://tip.golang.org/pkg/io/fs/) driver which is compatible with Go 1.16+ [file `//embed` directives](https://tip.golang.org/pkg/embed/).

Side note: I think all the documentation in this file would be better placed into a package-level godoc documentation comment, so that on https://pkg.go.dev this package shows up with package docs and usage examples similar to [this](https://pkg.go.dev/github.com/fluhus/godoc-tricks#pkg-overview) and [this](https://pkg.go.dev/github.com/fluhus/godoc-tricks#pkg-examples)

Feel free to ignore if this is not helpful :)